### ISSUE TITLE
Video previews not loading - causes failed login attempt

### ISSUE THREAD

**Issue by sjoshua270** [dermotduffy/frigate-hass-card#872]
I have a basic card set up:

    - type: custom:frigate-card
      cameras:
        - camera_entity: camera.front
          live_provider: ha
          frigate:
            url: https://<my_domain>/ccab4aaf_frigate-proxy/dashboard
      menu:
        style: outside

Whenever I try to look at clips or snapshots, the icons fail to load, and I get this notification in Home Assistant:

![image](https://user-images.githubusercontent.com/3036392/196005270-ccce3101-80a3-415a-9ecb-365826392880.png)

**Comment by dermotduffy** [comment]
* Please hold down your Frigate menu button and paste the resulting diagnostics here.
* Does Frigate work otherwise? When you use the HA media browser (not the card), can you see Frigate media?

**Comment by wvdeadhead** [comment]
> * Please hold down your Frigate menu button in the card and paste the resulting diagnostics here.
> * Does Frigate work otherwise? When you use the HA media browser (not the card), can you see Frigate media?

I have the same issue.  Frigate media is unavailable and causes hundreds of IP Bans when I try to access.

Holding down the Frigate menu button to get diagnostics does nothing... maybe it's due to my card's button configuration?

**Comment by dermotduffy** [comment]
@wvdeadhead Is it possible you're using an old card version? The current version is [v4.0.0](https://github.com/dermotduffy/frigate-hass-card/releases/tag/v4.0.0). Current version of the integration is [v3.0.0](https://github.com/blakeblackshear/frigate-hass-integration/releases/tag/v3.0.0). Make sure both are up to date and in sync. Your lack of diagnostics support, and the actual problem, might be related ;-)

**Comment by wvdeadhead** [comment]
> @wvdeadhead Is it possible you're using an old card version? The current version is [v4.0.0](https://github.com/dermotduffy/frigate-hass-card/releases/tag/v4.0.0). Current version of the integration is [v3.0.0](https://github.com/blakeblackshear/frigate-hass-integration/releases/tag/v3.0.0). Make sure both are up to date and in sync. Your lack of diagnostics support, and the actual problem, might be related ;-)

Thanks for the follow-up.  Both the integration and card are installed via HACS and confirmed to be on the latest versions.

**Comment by dermotduffy** [comment]
@wvdeadhead Hmm. Then I can't tell why diagnostics don't work. Can you paste your card configuration?

**Comment by wvdeadhead** [comment]
> @wvdeadhead Hmm. Then I can't tell why diagnostics don't work. Can you paste your card configuration?

Sure thing.  This is the config I had tried.  Maybe I mis-configured?

`       - type: custom:frigate-card
         cameras:
           - camera_entity: camera.backyard
             live_provider: webrtc-card
             webrtc_card:
               entity: camera.backyard_rtsp
         live:
           controls:
             thumbnails:
               mode: below
               size: 100px
               media: clips
           preload: true
           webrtc_card:
             ui: true`

**Comment by wvdeadhead** [comment]
I think I found the issue.

1) Browser cache
2) Removing the "px" from the thumbnails size.

All is working as expected now... including diagnostics.

**Comment by dermotduffy** [comment]
Thanks @wvdeadhead , glad you got it figured out.

@sjoshua270 Any development on your end? If not, would you mind [following these instructions](https://github.com/dermotduffy/frigate-hass-card/issues/872#issuecomment-1279827353) to provide more information?

**Comment by sjoshua270** [comment]
It looks like it may have been cache for me, as well.
I had restarted Home Assistant and was not able to show diagnostic output.

I tried on a new machine, and all my settings were applied with no issues.
Today, my mobile app is updated as well.

Thanks for the prompt help! I honestly got busy and forgot to check on this
issue, since it was minor for me.  Didn't realize I was using a cached
3.0.0 😂
